lib/mountable_file_server/metadata.rb

```diff
@@ -0,0 +1,45 @@
+require 'dimensions'
+
+module MountableFileServer
+  class Metadata
+    attr_reader :size, :content_type, :height, :width
+
+    def initialize(size:, content_type:, width: nil, height: nil)
```

Comment:
Is the syntax the prefered way to receive arguments in this code base or is this an exception here?